### PR TITLE
Add link from profiles overview to the front end

### PR DESCRIPTION
**Pull request by kallehauge**
Add a link from the Community Profiles overview to the front end.

### CODE REVIEW

File: web/modules/custom/dbcdk_community/src/Plugin/Block/ProfilesBlock.php [hunk 1]
```diff
@@ -253,6 +280,13 @@ protected function parseProfile(Profile $profile, array $columns) {
           ]);
           break;
 
+        // Generate community profile link based on a pattern defined on the
+        // settings page.
+        case 'community_link':
+          $community_site_url = Url::fromUri($this->urlGenerator->generate($profile));
+          $row[] = Link::fromTextAndUrl($this->t('Biblo.dk'), $community_site_url);
```
Comment (kallehauge) [inline web/modules/custom/dbcdk_community/src/Plugin/Block/ProfilesBlock.php]: @kasperg I use "Biblo.dk" as my text for the link to the front end here. Should we agree on some sort of standard? I can see we use `See on community site` on Flagged Content

Comment (kasperg) [inline web/modules/custom/dbcdk_community/src/Plugin/Block/ProfilesBlock.php]: We should agree on a standard. You decide which you prefer.

About url/link generation is calling these static methods the way to go? [It seems like the way to go compared to the complexity of using an injected LinkGenerator](https://github.com/DBCDK/biblo-admin/blob/develop/web/modules/custom/dbcdk_community/src/Plugin/Block/FlaggedContentList.php#L172-L175). These classes do not seem to hook into other parts of the Drupal architecture anyway.

Comment (kallehauge) [inline web/modules/custom/dbcdk_community/src/Plugin/Block/ProfilesBlock.php]: After verbal communication: We're changing to "View on Biblo.dk".

File: web/modules/custom/dbcdk_community/src/Plugin/Block/ProfilesBlock.php [hunk 2]
```diff
@@ -253,6 +280,13 @@ protected function parseProfile(Profile $profile, array $columns) {
           ]);
           break;
 
+        // Generate community profile link based on a pattern defined on the
+        // settings page.
+        case 'community_link':
+          $community_site_url = Url::fromUri($this->urlGenerator->generate($profile));
+          $row[] = Link::fromTextAndUrl($this->t('View on Biblo.dk'), $community_site_url);
```
Comment (kallehauge) [inline web/modules/custom/dbcdk_community/src/Plugin/Block/ProfilesBlock.php:287]: @kasperg I have changed the text as we discussed :)

**Comment by kasperg** [comment]
Great. If we also export the config for the profile pattern we should be good.

File: web/config/sync/dbcdk_community.settings.yml [hunk 3]
```diff
@@ -1,3 +1,4 @@
 community_service_url: ''
 community_site_post_url_pattern: '/posts/[id]'
 community_site_comment_url_pattern: '/comments/[id]'
+community_site_profile_url_pattern: '/profile/[id]'
```
Comment (kasperg) [inline web/config/sync/dbcdk_community.settings.yml]: Dette skal så også være `/profil/[id]`?

File: web/modules/custom/dbcdk_community/src/Plugin/Block/ProfilesBlock.php (hunk 2 above)
Comment (kasperg) [inline web/modules/custom/dbcdk_community/src/Plugin/Block/ProfilesBlock.php:287]: :+1:

**Comment by kasperg** [comment]
Merged.

**Pull request merged**